### ISSUE TITLE
[Keyboard Navigation - Makecode - Microbit Controls] Focus indicator is not visible on the tree View of Microbit Menu controls.

### ISSUE THREAD

**Issue by v-mesala** [Microsoft/pxt#2621]
**User Experience**
If Focus indicator is not visible on the tree view of the menu controls then the user with visual disabilities might be confuse where the focus is on the screen.

**Test Environment**
OS: RS2 Version 1703(OS Build 15063.2)
Platform: Edge.
Tool: Color contrast Analyzer

**Repro Steps**
1-Navigate to https://makecode.microbit.org/acc
2-Navigate to the Micro bit section element and select Code control given on it.
3-Navigate to menu controls and verify the issue.

**Actual Result**
Focus indicator is not visible on the tree View of Micro bit Menu controls.

**Expected Result**
Focus indicator should be visible on the tree View of Micro bit Menu controls.

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/Documents/LCA/MAS%20-%20July%201,%202017%20Release/MAS%202.4.7%20%E2%80%93%20Focus%20Visible%20.docx?d=w3b98b04f4fab4d3e81afa0906a34849c&csf=1

**Suggested fix**
1.Using user interface components that are highlighted by the user agent when they receive focus 
2.Using CSS to change the presentation of a user interface component when it receives focus  
3.Using the default focus indicator for the platform so that high visibility default focus indicators will carry ov  
4.Using an author-supplied, highly visible focus indicator  
5.Using script to change the background color or border of the element with focus  
#div a:hover, #div a:active, #div a:focus { 
  background-color: #DCFFFF; 
  color:#000066; 
} 
To provide an outline around the element we can use the below  
a { outline: none; }

**Refer Attachment**
[MAS 2.4.7_Makecode_Focus indicator is not visible.zip](https://github.com/Microsoft/pxt/files/1193305/MAS.2.4.7_Makecode_Focus.indicator.is.not.visible.zip)

**Comment by veler** [comment]
This is part of Blocky, a third-part component.

**Comment by abchatra** [comment]
We have got exception for Blockly bugs from Julia Liuson (CVP). Closing these as Won't Fix. Following is the reasoning.

----------------------

MakeCode can be used in either Text editor or Block editor.

Blocks editor is inherently visual and is based on google blockly. Usage requires drag and drop functionality and we don’t have a equivalence in MakeCode for this w.r.t accessibility. 
We have evaluated couple of research projects which tries to find a equivalence to block based coding for accessibility needs: 
•	Project Torino (Requires students to buy physical blocks)
•	Accessible blockly (This is text based coding without syntax errors)
Both these are drastic departure from the blockly and provides a complete new experience. These are still in research and we intend not to use them in MakeCode at this point of time.

As workaround, the text editor in MakeCode is the way we see students with the need of accessibility using MakeCode.
